src/styles/input.css

```diff
@@ -7,7 +7,9 @@
   font-size: var(--font-size);
 }
 
-.ui.input > input:focus {
+.ui.input > input:focus,
+.ui.form input[type='text']:focus,
+.ui.form textarea:focus {
```

Comment:
Nice catch on the `textarea` but let's add _all_ of the other Semantic selectors as well. I'm not sure why Semantic is styled like this, but I figure we should match it to cover our bases.

```
.ui.form input:not([type]):focus,
.ui.form input[type="date"]:focus,
.ui.form input[type="datetime-local"]:focus,
.ui.form input[type="email"]:focus,
.ui.form input[type="number"]:focus,
.ui.form input[type="password"]:focus,
.ui.form input[type="search"]:focus,
.ui.form input[type="tel"]:focus,
.ui.form input[type="time"]:focus,
.ui.form input[type="text"]:focus,
.ui.form input[type="file"]:focus,
.ui.form input[type="url"]:focus,
.ui.form textarea:focus {
  ...
}
```